src/pages/CarsList/CarsList.tsx

```diff
@@ -18,7 +18,7 @@ function CarsList() {
   useEffect(() => {
     window.scrollTo(0, 0);
   }, [location]);
-  
+
   const [carType, SetCarType] = useState<string>(
     ["sedan", "SUV", "hatchback", "coupe"].includes(
       JSON.parse(JSON.stringify(searchtext)).searchtext as string
```

Comment:
Why do we need this?
`JSON.parse(JSON.stringify(searchtext))`

Please remove such usages from everywhere. stringify and parse are expensive operations, and there is little sense in doing both in same line.